src/mame/heathkit/tlb.cpp

```diff
@@ -1045,7 +1045,8 @@ void heath_tlb_device::device_add_mconfig(machine_config &config)
 	// MC 6845 uses a character clock, divide the DOT clock by 8.
 	MC6845(config, m_crtc, BASE_DOT_CLOCK / 8);
 	m_crtc->set_screen("screen");
-	m_crtc->set_show_border_area(true);
+	m_crtc->set_show_border_area(false);
+	m_crtc->set_visarea_adjust(-30, 30, -5, 15);
```

Comment:
The values programmed by the ROM leaves a large area on the right and no padding at the top. Would it be better to show border, and just adjust max_x and min_y with negative values? I think that would actually help with another issue of resizing. There is line 25 that can be turned off and on. On the real system the ROM just changes R6 - displayed vertical to either 24 or 25. With this change, it causes the text to move, but I think with show border area on, it just added or removed a line, keeping everything else in place. I'll verify that behavior. Is there a different change you are suggesting?  

<img width="1494" alt="Screenshot 2024-02-04 at 10 24 04 AM" src="https://github.com/mamedev/mame/assets/8291090/12712801-4596-4240-a8d4-f6058f23bb96">

Reply:
The 6845 will either do show border, or the adjustments, and the show border takes precedence. Keeping it turned on will still have too little space at the top, and too much on the right. For comparison, this is what it looks like with my PR. On the real system, it had adjustments for the width/height/ x-offset/y-offset, and could be adjusted to look like the this capture. If there is a way to do with without `set_visarea_adjust()`, let me know, or point me to some code that does it.

<img width="1452" alt="Screenshot 2024-02-04 at 11 04 07 AM" src="https://github.com/mamedev/mame/assets/8291090/8af8dde4-b680-4acb-b1ab-cfae977b8259">

Reply:
Look, there must be a way by a 6845 to determine what's border and what's sync, what the `set_show_border_area` should really do is making sure that only former is shown. 
1. I didn't found anything noteworthy in the official 6845 datasheet, didn't try the others.
2. What I do on first time Sharp X1 setup is just setting horizontal / vertical size/position in my config to have some kind of sense. Why I've bothered back then is because ...
3. ... suikoden (X1) sets up a 640x8 visible area to print the Koei copy protection failure, and MAME is pretty well known to be unstable and unwatchable at illegal resolutions/refresh rates.
4. Frogger (BBC Micro) does heavy partial updates with the CRTC registers, so that each character gets delimited by this so called "screen visible area" that is not behaving as one;
5. Dezalis (X1) does a nasty trick during stages: it changes those values so to simulate raster zooming for the backplane while leaving the score display of the same size (currently MAME don't even draw the score).
6. Other misc stuff if I think about it (namely [this](https://github.com/mamedev/mame/blob/ba52f3124a5cb9b83ee5bc4422b40d7a2c493804/src/mame/dataeast/progolf.cpp#L11), have practically no experience with CPC at the time of this writing.

I feel like _on its own_ 6845 don't really sync stand-alone but needs an interface in between, but I don't have any proof to back this up.

Reply:
I'm occasionally seeing garbage at the top, so maybe this approach isn't the correct way to do it, or I need to clear some memory at startup. I haven't seen it occur on any initial runs, only occasionally if I go back to the UI, and select the system again, so it seems like reused memory that needs clearing.

   On a real system there are several potentiometers to control things like height, width, vertical offset and horizontal offset. I am just trying to do, is just show the screen like it would be on a real system which has been properly adjusted with the potentiometers - centered, filling the screen with some uniform buffer around the display area.

Reply:
Ok, I have reserves over what both `set_show_border_area` and `set_visarea_adjust` tries to accomplish here, and that's material for an issue. As it stands this PR can be applied lacking better options for now.

Reply:
@angelosa I removed the call to `set_visarea_adjust` and it is now using layout to add some padding.